docker/docker-compose-cockroach.yml

```diff
@@ -0,0 +1,52 @@
+
+version: '3'
+services:
+  cockroach:
+    hostname: cockroach
+    image: cockroachdb/cockroach
+    container_name: cockroach
+    tty: false
+    restart: "on-failure:5"
+    security_opt:
+     - "no-new-privileges"
+    cap_drop:
+    - ALL
+    command: ["start", "--insecure"]
+  statsd:
```

Comment:
We have removed statsd from base docker-compose. It is not a required dependency and makes the experience of local development through docker-compose much faster

Reply:
What are the used Go functions that require statsd?

Reply:
It's an optional dependency.  Temporal server emits metric using tally, so application will continue to emit metric.  It's just that we are not configuring a statsd collector to store all of the metic emitted by the application.